migrate_nidirect_node/src/EventSubscriber/PostMigrationSubscriber.php

```diff
@@ -126,6 +139,37 @@ public function onMigratePostImport(MigrateImportEvent $event) {
     if ($event_id == 'node_revision_landing_page') {
       $this->landingPageUpdates();
     }
+
+    // Ensure some redirects of interest are always present.
+    $this->redirects();
+  }
+
+  /**
+   * Guarantee certain redirects exist after each migration.
+   */
+  protected function redirects() {
+    $redirects = [
+      'contacts/contacts-az/police-service-northern-ireland' => 'internal:/node/77',
+      'contacts/contacts-az/migrant-help' => 'internal:/node/166',
```

Comment:
Johan will this be the mechanism to fix all links to contacts? Aside from these two examples, there are likely to be hundreds more ...

Reply:
Absolutely right, I thought it was just for two items for some reason. Latest commit adds a process to cover them all in one go.